server/data/accreditedProgrammesApi/referralClient.test.ts

```diff
@@ -93,8 +89,8 @@ pactWith({ consumer: 'Accredited Programmes UI', provider: 'Accredited Programme
 
     beforeEach(() => {
       provider.addInteraction({
-        state: `Referral summaries exist for an organisation with the ID ${organisationId}`,
-        uponReceiving: `A request for the summaries for an organisation with the ID ${organisationId}`,
+        state: 'Referral summaries exist for organisation a026cc07-8e0b-40dd-9b66-1b3dacecc63d',
```

Comment:
Organisation IDs are the same as Prison IDs in our service (so three characters), and the referral is fetched through the Offering so for this could be `BWN` to match our test data, rather than this UUID.